app/src/main/java/org/schabi/newpipe/util/FireTvUtils.java

```diff
@@ -1,10 +1,31 @@
 package org.schabi.newpipe.util;
 
+import android.annotation.SuppressLint;
+import android.content.pm.PackageManager;
+
+import android.view.KeyEvent;
 import org.schabi.newpipe.App;
 
 public class FireTvUtils {
+    @SuppressLint("InlinedApi")
     public static boolean isFireTv(){
         final String AMAZON_FEATURE_FIRE_TV = "amazon.hardware.fire_tv";
-        return App.getApp().getPackageManager().hasSystemFeature(AMAZON_FEATURE_FIRE_TV);
+
+        PackageManager pm =  App.getApp().getPackageManager();
+
+        return pm.hasSystemFeature(AMAZON_FEATURE_FIRE_TV)
+                || pm.hasSystemFeature(PackageManager.FEATURE_LEANBACK);
```

Comment:
I guess all Android TVs are now identified as FireTV. Is this correct?
Maybe rename this class to TV or device utils and create a new method `isTV()`.

Reply:
> Maybe rename this class to TV or device utils and create a new method isTV()

That's probably a correct course. I didn't want to rename the existing method, because such refactoring always causes a lot of conflicts during rebases, and I'd like to keep this PR easy to rebase.

Can we postpone this until this is merged? You can rename it yourself, or I will send a separate PR with just that change.

Reply:
The method was only used once before you changed it. That's why I guess there should not be complex merge conflicts.

Reply:
Renamed it.